node/builder.go

```diff
@@ -371,6 +371,9 @@ func ConfigCommon(cfg *config.Common) Option {
 				Override(new(dtypes.BootstrapPeers), modules.ConfigBootstrap(cfg.Libp2p.BootstrapPeers)),
 			),
 		),
+		Override(AddrsFactoryKey, lp2p.AddrsFactory(
```

Comment:
@magik6k thoughts on this? We already have a place where we set the `AddrsFactory` up in the libp2p option, but that place doesnt have access to the config

Reply:
That's ok, in theory it should be possible to build a functioning node without config, so that one just sets the default